lms/templates/ccx/schedule.html

```diff
@@ -54,44 +56,62 @@
 </section>
 
 <div class="ccx-schedule-sidebar">
-  <div class="ccx-sidebar-panel" id="dirty-schedule">
-    <h2>${_('Save changes')}</h2>
-    <form role="form">
-      <p>${_("You have unsaved changes.")}</p>
+  <div class="ccx-sidebar-panel" id="dirty-schedule" tabindex="-1" role="region"
+       aria-labelledby="ccx_schedule_save_changes_heading">
+    <h2 id="ccx_schedule_save_changes_heading">${_('Save changes')}</h2>
+    <form>
+      <p id="message_save">${_("You have unsaved changes.")}</p>
       <div class="field">
         <br/>
-        <button id="save-changes">${_("Save changes")}</button>
+        <button id="save-changes" aria-describedby="message_save">${_("Save changes")}</button>
       </div>
     </form>
   </div>
-  <div class="ccx-sidebar-panel" id="ajax-error">
+  <div class="ccx-sidebar-panel" id="ajax-error" tabindex="-1" role="region" aria-labelledby="ccx_schedule_error_message">
     <h2>${_('Error')}</h2>
-    <p>${_("There was an error saving changes.")}</p>
+    <p id="ccx_schedule_error_message">${_("There was an error saving changes.")}</p>
   </div>
-  <div class="ccx-sidebar-panel">
-    <h2>${_('Schedule a Unit')}</h2>
+  <div class="ccx-sidebar-panel" aria-labelledby="ccx_schedule_unit"
+       id="ccx_schedule_unit_panel" role="region">
+    <h2 id="ccx_schedule_unit">${_('Schedule a Unit')}</h2>
     <form role="form" id="add-unit" name="add-unit" class="ccx-form">
       <div class="field">
-        <b>${_('Section')}</b><br/>
-        <select name="chapter"></select>
+        <label for="ccx_chapter"><b>${_('Section')}</b></label>
+        <select name="chapter" id="ccx_chapter" ></select>
       </div>
       <div class="field">
-        <b>${_('Subsection')}</b><br/>
-        <select name="sequential"></select>
+        <label for="ccx_sequential"><b>${_('Subsection')}</b></label>
+        <select name="sequential" id="ccx_sequential"></select>
       </div>
       <div class="field">
-        <b>${_('Unit')}</b><br/>
-        <select name="vertical"></select>
+        <label for="ccx_vertical"><b>${_('Unit')}</b></label>
+        <select name="vertical" id="ccx_vertical"></select>
       </div>
       <div class="field datepair">
-        <b>${_('Start Date')}</b><br/>
-        <input placeholder="Date" type="date" class="date" name="start_date"/>
-        <input placeholder="time" type="time" class="time" name="start_time"/>
+        <!-- Translators: This explains to people using a screen reader how to interpret the format of YYYY-MM-DD -->
```

Comment:
This needs to be directly above the line with the translation string so you need to move it to right above L95. 

Also if this is a Mako file, your syntax is wrong. See http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html#mako-template-files